README.md

```diff
@@ -27,3 +27,14 @@ This plugin allows `fluentd` to send data to a remote syslog endpoint like [pape
 * `SYSLOG_PORT_2=52232`
 
 You can also set `SYSLOG_HOST` and `SYSLOG_PORT`..
+
+### Deis Output
+Deis output is a custom fluentd plugin that was written to forward data directly to deis components while filtering out data that we did not care about. We have 2 pieces of information we care about currently.
+
+1) Logs from applications that are written to stdout within the container and the controller logs that represent actions against those applications. These logs are sent to an internal messaging system ([NSQ](http://nsq.io)) on a configurable topic. The logger component then reads those messages and stores the data in an ring buffer.
+
+2) Metric data from the nginx based router. We take the log and parse out `request_time`, `response_time`, and `bytes_sent`. Each one of these metrics makes up a series that we will ultimately send to our InfluxDB system. Attached to each series is the host the data came from (where router is running) and the status code for that request.
+
+The topics these messages are put on are configurable via environment variables.
```

Comment:
I just looked at the [GH rendering of this file](https://github.com/jchauncey/fluentd/blob/4cea03a80d4ded4fd63d4a9971d711989f33b41d/README.md) and things look fine, so nevermind. I guess GH flavored markdown knows what it's doing!